game-core/src/main/java/games/strategy/triplea/ui/TripleAFrame.java

```diff
@@ -227,9 +227,8 @@ public static TripleAFrame create(final IGame game, final LocalPlayers players)
     uiContext.setLocalPlayers(players);
 
     final TripleAFrame frame = Interruptibles
-        .awaitResult(() -> SwingAction.invokeAndWaitResult(() -> new TripleAFrame(game, players, uiContext)))
-        .result
-        .get();
+        .awaitResult(() -> SwingAction.invokeAndWaitResult(() -> new TripleAFrame(game, players, uiContext))).result
```

Comment:
Formatting nit, please keep `.result` on its own line.  `.get()` should be on same line as `.result` (but that is an existing problem)

Reply:
Updated. The formatter doesn't seem to like `.result` on its on line and joins it even though I have 'never join lines' on. Formatted it slightly differently now.